var/spack/repos/builtin/packages/grib-api/package.py

```diff
@@ -35,15 +35,20 @@ class GribApi(Package):
 
     version('1.17.0', 'bca7114d2c3100501a08190a146818d2')
 
+    variant('jpeg', default=True)
```

Comment:
Can you add a description to this variant? It can be something as simple as:
```python
variant('jpeg', default=True, description='Enable JPEG support')
```